Common.mk

```diff
@@ -456,10 +457,10 @@ validate-checksums: $(BINARY_TARGETS)
 helm/build: $(OUTPUT_DIR)/ATTRIBUTION.txt
 helm/build: $(if $(filter true,$(REPO_NO_CLONE)),,$(HELM_GIT_CHECKOUT_TARGET))
 helm/build: $(if $(wildcard $(MAKE_ROOT)/helm/patches),$(HELM_GIT_PATCH_TARGET),)
-	HELM_REGISTRY=$(IMAGE_REPO) \
 	IMAGE_TAG=$(IMAGE_TAG) \
+	HELM_IMAGE_LIST=$(HELM_IMAGE_LIST) \
```

Comment:
Dont worry about changing this now, but something for the future.  This call is a bit out of hand considering its setting env vars and passing in a number of args.  No real suggestions, just a note.